scripts/genpinctrl/genpinctrl.py

```diff
@@ -429,7 +425,7 @@ def get_mcu_signals(cube_path, gpio_ip_afs):
                 elif family == "STM32F1":
                     found_afs = [0]
                 else:
-                    continue
+                    found_afs = [None]
```

Comment:
Would probably be good to emphasize that this `else` is for `analog` mode, it's less clear now.

Reply:
Added a comment